src/OpenTK/Platform/X11/API.cs

```diff
@@ -52,6 +52,13 @@ namespace OpenTK.Platform.X11
 
     #endregion
 
+    // X11 has some defined values
+    internal static class Consts
+    {
+        public static readonly IntPtr None = IntPtr.Zero;
+        public static readonly IntPtr CurrentTime = IntPtr.Zero;
+    }
+
```

Comment:
Hahaha, At least I'm not the only confused one. Just add a member comment explaining that these are defined like this, because it's the same way they're defined in the X.h header.